### PR TITLE
Feature/bxb 3761   move zookeeper library from at deps to media deps develop fix

### PR DESCRIPTION
**Pull request by alonzno**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Copyright Airtime Media 2021
+project(at_zookeeper_c)
+cmake_minimum_required(VERSION 3.12.0)
+
```
Comment (jim-wong) [inline CMakeLists.txt]: Move this file to an `airtime` subdirectory

**Comment by jim-wong** [comment]
Why are changes other than CMakeLists.txt changing?

**Comment by alonzno** [comment]
@jim-wong Some of the zookeeper code was in `at-deps/third_party/zookeeper`, in particular everything that is now in the top level `generated` folder.  That is why I had adjusted the directory structure originally to separate generated and source code, it was to match the structure of `at-deps/third_party/zookeeper`

**Review by sumitkandpal — APPROVED** [review]

File: PATCHES [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ 
++// airtime start
++#ifdef htonll  // Mac's arpa/inet.h defines this
++# undef htonll
++#endif  // htonll
++// airtime end
+ int64_t htonll(int64_t v);
+ 
+ #ifdef __cplusplus
+
```
Comment (jim-wong) [inline PATCHES]: Move this to an airtime-specific directory?

**Review by jim-wong — CHANGES_REQUESTED** [review]
See comments

File: PATCHES (hunk 2 above)
Comment (alonzno) [reply PATCHES]: Moved

File: CMakeLists.txt (hunk 1 above)
Comment (alonzno) [reply CMakeLists.txt]: Moved

**Comment by alonzno** [comment]
@jim-wong Good now?

**Review by jim-wong — APPROVED** [review]

**Pull request merged**